Code/src/main/java/nl/utwente/viskell/ui/components/BlockContainer.java

```diff
@@ -15,4 +17,25 @@
      */
     public void handleConnectionChanges(boolean finalPhase);
 
+    /** Gets the value of the property boundsInLocal. */
+    public Bounds getBoundsInLocal();
+    
+    /** Transforms a point from the local coordinate space of this Node into the coordinate space of its scene. */
+    public Bounds localToScene(Bounds bounds);
```

Comment:
Aren't `localToScene` and `getBoundsInLocal` JavaFX methods that every Node gets anyway?